Sources/Plasma/Apps/plUruLauncher/plClientLauncher.h

```diff
@@ -70,8 +70,16 @@ class plClientLauncher
         kRepairGame = kHaveSelfPatched | kClientImage | kGameDataOnly,
     };
 
+    enum NetCoreState
+    {
+        kNetCoreInactive,
+        kNetCoreActive,
+        kNetCoreShutdown,
+    };
+
     uint32_t   fFlags;
     plFileName fServerIni;
+    uint8_t    fNetCoreState;
```

Comment:
Why not just declare it `NetCoreState` and let the compiler warn on funny values?